dace/sdfg/nodes.py

```diff
@@ -877,6 +880,7 @@ class Map(object):
     # List of (editable) properties
     label = Property(dtype=str, desc="Label of the map")
     params = ListProperty(element_type=str, desc="Mapped parameters")
+    param_types = DictProperty(key_type=str, value_type=str, desc="Types of mapped parameters")
```

Comment:
shouldn't param types have a typeclass value?

Reply:
I had saved the typestrings set by the transformation as a Dict(str,str), such that if a user wants to give the type info as a part of a transformation then the user can just write the str, but in a sense, having the type be a typeclass has information value in the "param_types" parameter, I will update.